components/brave_new_tab_ui/components/default/vpn/vpn_card.tsx

```diff
@@ -0,0 +1,139 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+import { useDispatch } from 'react-redux'
+
+import Toggle from '@brave/leo/react/toggle'
+import { getLocale } from '$web-common/locale'
+import * as S from './styles'
+import VPNShieldsConnecting from './vpn-shields-connecting'
+import * as Actions from '../../../actions/brave_vpn_actions'
+import { ConnectionState, Region } from '../../../api/braveVpn'
+
+export const VPNWidgetTitle = () => {
+  return (
+    <>
+      <S.HeaderIcon name='product-vpn' />
+      {getLocale('braveVpnWidgetTitle')}
+    </>
+  )
+}
+
+export const VPNWidgetHeader = () => {
+  return (
+    <S.StyledTitle>
+      <VPNWidgetTitle />
+    </S.StyledTitle>
+  )
+}
+
+export const VPNPromoWidget = () => {
+  const dispatch = useDispatch()
+
+  const featureList = React.useMemo(
+    () => [
+      getLocale('braveVpnFeature1'),
+      getLocale('braveVpnFeature2'),
+      getLocale('braveVpnWidgetFeature3')
+    ],
+    []
+  )
```

Comment:
nit: These are always the same, so you can lift them to outside the component:

```ts
const featureList = [ ... ]
```

Reply:
As an added optimization, you could generate this outside the component:

```ts
const featureList = <>
    {[
      getLocale('braveVpnFeature1'),
      getLocale('braveVpnFeature2'),
      getLocale('braveVpnWidgetFeature3')
    ].map(entry => <S.SellingPoint key={entry}>
        <S.SellingPointIcon name='shield-done' />
        <S.SellingPointLabel>{entry}</S.SellingPointLabel>
    </S.SellingPoint>)}
</>
```

as it won't ever change